fe/src/project/Seller.Portal/areas/Inventory/components/InventoryAddForm/InventoryAddForm.js

```diff
@@ -72,7 +72,7 @@ const InventoryAddForm = (props) => {
                 return res.json().then(jsonRes => {
                     if (res.ok) {
                         toast.success(jsonRes.message);
-                        setShowBackToInventoryListButton(true);
+                        setDisableSaveButton(true);
```

Comment:
**suggestion:** We should get rid of it. We have already the disable property. It's not present in many other places like ClientsForm above

Reply:
@dawiddworak88 

I know that I can operate on the id in useForm, but think about the PR related to displaying the id, even after saving I have to use this state because I will not display information about the id